exchanges/bybit/bybit.go

```diff
@@ -612,7 +612,8 @@ func (by *Bybit) CancelTradeOrder(ctx context.Context, arg *CancelOrderParams) (
 // GetOpenOrders retrieves unfilled or partially filled orders in real-time. To query older order records, please use the order history interface.
 // orderFilter: possible values are 'Order', 'StopOrder', 'tpslOrder', and 'OcoOrder'
 func (by *Bybit) GetOpenOrders(ctx context.Context, category, symbol, baseCoin, settleCoin, orderID, orderLinkID, orderFilter, cursor string,
-	openOnly, limit int64) (*TradeOrders, error) {
+	openOnly, limit int64,
+) (*TradeOrders, error) {
```

Comment:
Suggestion: for all these changes is to just one liner them until we get an exact line length standard on project